prokube-ai/kubeflow-examples
# Llama2 chat deployment in kserve

This Pull Request introduces the code necessary for deploying llama2 models utilizing kserve. Given that there is no existing kserve implementation for Hugging Face transformers, we've employed custom kserve predictors to achieve this functionality.

How to Test:

See if you're able to deploy it on your own by following the detailed steps provided in the README. Feedback and commits are very welcome :).
